openassessment/templates/openassessmentblock/staff_area/oa_student_info.html

```diff
@@ -0,0 +1,316 @@
+                Viewing learner: {{ learner }}
+            {% endblocktrans %}
+        </span>
+    </h2>
+
+    <div class="staff-info__status ui-staff__content__section wrapper--ui--collapse staff-info__student__response">
+        <div class="ui-staff ui-toggle-visibility is--collapsed">
+            <h2 class="staff-info__title ui-staff__subtitle ui-toggle-visibility__control">
+                <i class="icon fa fa-caret-right" aria-hidden="true"></i>
+                <span>{% trans "Learner's Response" %}</span>
+            </h2>
+            <div class="ui-toggle-visibility__content">
+                {% if workflow_cancellation %}
+                    <p>
+                        {% blocktrans with removed_by_username=workflow_cancellation.cancelled_by removed_datetime=workflow_cancellation.cancelled_at|utc|date:"F j, Y H:i e" %}
+                            Learner submission removed by {{ removed_by_username }} on {{ removed_datetime }}
+                        {% endblocktrans %}
+                    </p>
+                    <!-- Comments: Reason for Cancellation-->
+                    <p>
+                        {% blocktrans with comments=workflow_cancellation.comments %}
+                            Comments: {{ comments }}
+                        {% endblocktrans %}
+                    </p>
+                {% else %}
+                    {% include "openassessmentblock/oa_submission_answer.html" with answer=submission.answer answer_text_label="The learner's response to the question above:" %}
+                {% endif %}
+
+                {% if submission.file_url %}
+                    <a href="{{ submission.file_url }}" class="submission--file">
+                        {% trans "The file associated with this response." %}
+                    </a>
+                    <span>{%  trans "Caution: This file was uploaded by another course learner and has not been verified, screened, approved, reviewed, or endorsed by edX. If you decide to access it, you do so at your own risk." %}</span>
+                {% endif %}
+            </div>
+        </div>
+    </div>
+
+    {% if peer_assessments != None %}
+        <div class="staff-info__status ui-staff__content__section wrapper--ui--collapse">
+            <div class="ui-staff ui-toggle-visibility is--collapsed">
+                <h2 class="staff-info__title ui-staff__subtitle ui-toggle-visibility__control">
+                    <i class="icon fa fa-caret-right" aria-hidden="true"></i>
+                    <span>{% trans "Peer Assessments for This Learner" %}</span>
+                </h2>
+                <div class="ui-toggle-visibility__content">
+                    {% if peer_assessments %}
+                        {% for assessment in peer_assessments %}
+                            {% with peer_num=forloop.counter %}
+                                <h4 class="title--sub"> {% trans "Peer" %} {{ peer_num }}: </h4>
```

Comment:
The comment on this got lost--

Not introduced by us, but this is certainly bad for 18n. It should be a blocktrans with placeholder.